jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/websocket/WebSocketDiagnosticsCollector.java

```diff
@@ -92,6 +93,7 @@ public void collectDiagnostics(ICompilationUnit unit, List<Diagnostic> diagnosti
                             WebSocketConstants.RAW_ON_OPEN_PARAM_OPT_TYPES, unit, diagnostics);
                     // PathParam URI Mismatch Warning Diagnostic
                     uriMismatchWarningCheck(type, diagnostics, unit);
+                    serverEndpointWarningCheck(type, diagnostics, unit);
```

Comment:
again, just for clarity, if you could add a comment above this call to separate this diagnostic from the URI mismatch diagnostic, it would be helpful